crowbar_framework/lib/crowbar/upgrade_status.rb

```diff
@@ -130,5 +153,9 @@ def upgrade_steps_6_7
         :finished
       ]
     end
+
+    def lock_path
+      "/opt/dell/crowbar_framework/tmp/upgrade_status_lock"
```

Comment:
So we can pass it to `Crowbar::Lock::LocalBlocking`. If non-rails application is using the locking library (via UpgradeProgress), it fails in `Crowbar::Lock` when trying to create default path...